openapi/openapi/src/test/scala/endpoints/openapi/ReferencedSchemaTest.scala

```diff
@@ -1,17 +1,14 @@
 package endpoints.openapi
 
 import endpoints.openapi.model._
-import endpoints.{algebra, generic, openapi}
+import endpoints.{algebra, macros, openapi}
 import org.scalatest.{Matchers, WordSpec}
 
 class ReferencedSchemaTest extends WordSpec with Matchers {
 
   sealed trait Storage
-
-  object Storage {
-    case class Library(room: String, shelf: Int) extends Storage
-    case class Online(link: String) extends Storage
-  }
+  case class StorageLibrary(room: String, shelf: Int) extends Storage
```

Comment:
Having classes defined inside companion object caused `StackOverflowError ` in the compiler. It may be related to this bug: https://github.com/scala/bug/issues/11157